Dockerfile.release.old_cpu

```diff
@@ -0,0 +1,61 @@
+FROM golang:1.21-alpine as build
```

Comment:
This is the line that will allow to run this image in old CPUs ?

Reply:
No @vadmeste `ubi8` - ubi9 doesn't run on CPUs from 15yrs ago :-)